chainstate/src/rpc/types/block.rs

```diff
@@ -15,22 +15,28 @@
 
 use chainstate_types::BlockIndex;
 use common::{
-    chain::Block,
+    chain::{Block, GenBlock},
     primitives::{BlockHeight, Id, Idable},
 };
 
 #[derive(Debug, Clone, serde::Serialize)]
 pub struct RpcBlock {
     id: Id<Block>,
+    prev_block_id: Id<GenBlock>,
```

Comment:
minor but why store `prev_block_id` and `transaction_count_in_block` when they can be accessed from `block` at any moment with no cost?

Reply:
This is not storage. This is rpc response. This is what users will see.